ui/src/edit.tsx

```diff
@@ -1,14 +1,14 @@
 import CriteriaHolder from "criteriaHolder";
-import { useGroupAndCriteria } from "hooks";
+import { useGroupSectionAndGroup } from "hooks";
 import { getCriteriaPlugin, getCriteriaTitle } from "./cohort";
 
 export default function Edit() {
-  const { criteria } = useGroupAndCriteria();
-  const plugin = getCriteriaPlugin(criteria);
+  const { group } = useGroupSectionAndGroup();
+  const plugin = getCriteriaPlugin(group.criteria[0]);
 
   return (
     <CriteriaHolder
-      title={getCriteriaTitle(criteria, plugin)}
+      title={getCriteriaTitle(group.criteria[0], plugin)}
```

Comment:
For now I opted to always put the primary criteria at index zero rather than use the tags since it was easier in the short term. So this is generating the title based on the primary criteria.